101-point-to-site/azuredeploy.json

```diff
@@ -0,0 +1,171 @@
+{  
+   "$schema":"http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+   "contentVersion":"1.0.0.0",
+   "parameters":{  
+      "location":{  
```

Comment:
If this isn't different from the resource group location, then the new recommendation is to make a variable called location and it's value is resourceGroup().location. Otherwise, the users have to pick a location twice in the "Deploy to Azure" xp